### DISCUSSION TITLE
Question about the pager: "show all" removes select

### DISCUSSION THREAD

**Discussion by michabbb** [Power-Components/livewire-powergrid#628] [Q&A]
hi there,

it´s the first time i am using powergrid in a project and it´s very cool 😎 
but one thing is strange to me: 

as soon as you choose "all" (value => 0) from the drop-down (`footer.perPageValues`) the drop-down disappears.
and I guess it´s because if this:

```php
count(data_get($setUp, 'footer.perPageValues')) > 1
```

https://github.com/Power-Components/livewire-powergrid/blob/a4966bef3faef4541606d8e9983bbc512af14323/resources/views/components/frameworks/tailwind/footer.blade.php#L9

and as soon as the user has chosen to see all entries, he cannot go back to any other value, because the "perpage" dropdown is gone. 
am I missing something here or is that really intended to be that way? 😏 

I already figured out how to avoid that "disappearing":  I can set a fixed list of values:

```php
return [
            Header::make()->showSearchInput(),
            Footer::make()
                ->showPerPage(10,[10, 25, 50, 100, 500,1000])
                ->showRecordCount()
        ];
```

but that still does not help in case when I want to offer "0" (show all) and still keep the dropdown so the user is able to go back to 10 (for example). 🤷 

**THANKS**

**Comment by luanfreitasdev** [comment]
Hi, thanks for reporting this issue.

Can you send a PR with your solution?

**Reply by michabbb** [reply]
first - as asked - i would like to know what the "wanted" behavior here should be. maybe this isn´t an issue because somebody wanted it the way it is now 😏 I don´t know.... 🤷

**Reply by Unisolgr14** [reply]
Perhaps It is preferable if we have the option to choose again with different values.

**Reply by michabbb** [reply]
first of all i would be happy to know if - that what I described - really is the "normal" behavior (as it works right now) - or if I am doing something wrong. as I see the code (in case I have looked at the right place) - it looks to me that is "wanted", but if that is really the case, I am confused, why someone does it that way. so before I offer any kind of PR I wanted to find out:

- is it a bug ?
- is it wanted ?
- am I doing something wrong on my side ?

thanks

**Reply by Epitaffio** [reply]
We are also experiencing this strange behaviour
